apps/frontend/src/app/2_molecules/ConnectWalletButton/ConnectWalletButton.tsx

```diff
@@ -30,6 +37,21 @@ export const ConnectWalletButton: FC<
 }) => {
   const { t } = useTranslation();
   const [isOpen, toggle] = useReducer(state => !state, false);
+  const { addNotification } = useNotificationContext();
+
+  const onCopyAddress = useCallback(() => {
+    addNotification(
+      {
+        type: NotificationType.success,
+        title: t(translations.copyAddress),
+        content: '',
+        dismissible: true,
+        id: Math.floor(Math.random() * 1000),
+      },
+      5000,
```

Comment:
Since you used this in multiple places as well and we want to have things standardized, extract it a const `DEFAULT_NOTIFICATION_TIMEOUT` and use that instead.